README.md

```diff
@@ -50,7 +50,7 @@
 </p>
 
 <p align="center">
- <a href="https://www.erxes.io/![Cover](https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg)"><img src="https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg" width="100%" alt="erxes: Free and open fair-code licensed experience operating system (XOS)">
+ <a href="https://erxes.io/static/images/home/home1.png"><img src="https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg" width="100%" alt="erxes: Free and open fair-code licensed experience operating system (XOS)">
```

Comment:
@Pragya2056  please change this link to https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg

Reply:
I made the change